lib/client.js

```diff
@@ -112,16 +112,16 @@ Client.prototype.send = function() {
   // Respect IRC line length limit of 512 chars
   // https://tools.ietf.org/html/rfc2812#section-2.3
   if (message.length > 512) {
-    var parts   = message.split(':');
-    var command = parts[1];
-    var payload = parts[2];
+    var payload_marker  = message.indexOf(' :');
+    var command = message.substr(0, payload_marker);
+    var payload = message.substr(payload_marker + 2);
 
     var chunks = payload.split(' ');
```

Comment:
Although this is a kinda separate thing, if we want to respect the 512 character limit, we should still split messages that have no spaces, `Array(513).join('a') + ':' + Array(250).join('b')`

Not necessary to change it in this PR but bonus points :heart: